src/components/StakingVaults/EarnOpportunities.tsx

```diff
@@ -1,20 +1,22 @@
 import { ArrowForwardIcon } from '@chakra-ui/icons'
 import { Box, Button, HStack } from '@chakra-ui/react'
-import { AssetId, fromAssetId } from '@shapeshiftoss/caip'
+import { AssetId, ethAssetId, fromAssetId } from '@shapeshiftoss/caip'
 import {
   EarnOpportunityType,
   useNormalizeOpportunities,
 } from 'features/defi/helpers/normalizeOpportunity'
+import { foxAssetId, foxEthLpAssetId } from 'features/defi/providers/fox-eth-lp/const'
```

Comment:
Rename this to `constants` in a follow-up PR to match our module naming convention

Reply:
Resolving all comments to be able to merge the PR @stackedq